ui/app/components/database-role-edit.hbs

```diff
@@ -194,4 +194,23 @@
       </div>
     </form>
   </div>
+{{/if}}
+
+{{#if this.saveIssuerWarning}}
+  <Hds::Modal @color="warning" @onClose={{fn (mut this.saveIssuerWarning) ""}} data-test-issuer-warning as |M|>
+    <M.Header @icon="alert-circle">
+      Are you sure?
+    </M.Header>
+    <M.Body>
+      <p class="has-bottom-margin-m has-line-break" data-test-issuer-warning-message>
```

Comment:
The spacing on this makes it look a little odd (to me.) If you remove the classes, I think it looks a little cleaner. If the designs are saying something else, let me know and feel free to ignore me!
```suggestion
      <p data-test-issuer-warning-message>
```
<img width="578" alt="Screenshot 2025-03-31 at 2 22 05 PM" src="https://github.com/user-attachments/assets/4c34e80c-dbf7-4684-b7e3-dc82fb811b0e" />